app/controllers/notifications_controller.rb

```diff
@@ -75,6 +75,11 @@ def index
     check_out_of_bounds(scope)
 
     @total = scope.count
+
+    @inbox_notification    = current_user.notifications.inbox.count
+    @starred_notification  = current_user.notifications.starred.count
+    @archived_notification = current_user.notifications.archived.count
+
```

Comment:
Infact, ignore that, can we only show the inbox count and drop the other two? Archived and Starred will only ever go up!